client/Wayland/wlf_input.c

```diff
@@ -176,7 +176,7 @@ BOOL wlf_handle_pointer_axis(freerdp* instance, const UwacPointerAxisEvent* ev)
 	 * positive: 0 ... 0xFF  -> slow ... fast
 	 * negative: 0 ... 0xFF  -> fast ... slow
 	 */
-	step = abs(direction);
+	step = 8 * abs(direction);
```

Comment:
If you do this it would be a good idea to replace `direction = wl_fixed_to_int(ev->value);` with `direction = wl_fixed_to_double(ev->value);` in code above.